recipes-cockpit/cockpit/cockpit_145.bb

```diff
@@ -30,6 +31,7 @@ FILES_${PN} += "${libdir}/firewalld \
                 ${libdir}/security \
                 ${datadir}/appdata \
                 ${systemd_unitdir}/system/${PN}.socket \
+                ${datadir}/cockpit \
```

Comment:
this should not be needed since it's already the default and we are just appending with `+=`